packages/react-magma-dom/src/components/Popover/Popover.stories.tsx

```diff
@@ -0,0 +1,959 @@
+              {errorState.uppercase ? (
+                <CloseIcon
+                  color={args.isInverse ? '#FAAEB0' : '#D32821'}
+                  size={16}
+                />
+              ) : (
+                <DoneIcon
+                  color={args.isInverse ? '#ACF0C1' : '#178037'}
+                  size={16}
+                />
+              )}
+
+              <span
+                style={{
+                  color: errorState.uppercase
+                    ? args.isInverse
+                      ? '#FAAEB0'
+                      : '#D32821'
+                    : args.isInverse
+                    ? '#ACF0C1'
+                    : '#178037',
+                }}
+              >
+                Includes uppercase letter
+              </span>
+            </div>
+            <div style={{ display: 'flex', gap: '8px', alignItems: 'center' }}>
+              {errorState.specialSymbol ? (
+                <CloseIcon
+                  color={args.isInverse ? '#FAAEB0' : '#D32821'}
+                  size={16}
+                />
+              ) : (
+                <DoneIcon
+                  color={args.isInverse ? '#ACF0C1' : '#178037'}
+                  size={16}
+                />
+              )}
+
+              <span
+                style={{
+                  color: errorState.specialSymbol
+                    ? args.isInverse
+                      ? '#FAAEB0'
+                      : '#D32821'
+                    : args.isInverse
+                    ? '#ACF0C1'
+                    : '#178037',
+                }}
+              >
+                Includes special symbol
+              </span>
+            </div>
+          </div>
+        </PopoverContent>
+      </Popover>
+    </Card>
+  );
+};
+
+export const InputExample = InputExampleTemplate.bind({});
+InputExample.args = {
+  hasPointer: false,
+  width: 'target',
+};
+
+const DisabledTriggerExample = args => {
+  return (
+    <Card
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        height: '300px',
+        justifyContent: 'center',
+      }}
+      isInverse={args.isInverse}
+    >
+      <Popover {...args}>
+        <PopoverTrigger />
+        <PopoverContent>
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: '8px',
+            }}
+          >
+            <span style={{ fontWeight: 600 }}>Lorem ipsum dolor</span>
+            <span>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam.
+            </span>
+          </div>
+        </PopoverContent>
+      </Popover>
+    </Card>
+  );
+};
+
+export const DisabledTrigger = DisabledTriggerExample.bind({});
+DisabledTrigger.args = {
+  width: 320,
+  isDisabled: true,
+};
+
+const ProfileExampleTemplate = args => {
```

Comment:
This made me laugh and I love it so much 💯 
![image](https://github.com/user-attachments/assets/b4da3e27-f9d2-439a-935f-b69cc2414a52)